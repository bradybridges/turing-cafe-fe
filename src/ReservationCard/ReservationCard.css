h2 {
  color: white;
  text-shadow: 5px -3px 9px black;
}

p {
  border-bottom: solid 1px white;
  border-top: solid 1px white;
  color: white;
  padding: 10px 0px;
}

.resButton {
  background-color: #ddd;
  border-radius: 2px;
  height: 40px;
  margin-bottom: 10px;
  width: 90%;
}

.resButton:hover {
  background-color: teal;
  border: solid #000000a1 1px;
  box-shadow: 4px -4px 12px #0000009c;
  color: white;
}

.resCard {
  background-color: #2d3e39f0;
  border-radius: 4px;
  margin: 10px 0px;
  width: 32%;
}

