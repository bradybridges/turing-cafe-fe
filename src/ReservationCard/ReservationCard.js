import React from 'react';
import './ReservationCard.css'

const ReservationCard = (props) => {
  return (
    <section className='resCard'>
      <h2>{props.name}</h2>
      <p>Date: {props.date}</p>
      <p>Time: {props.time}</p>
      <p>Number of guests: {props.number}</p>
      <button id={props.id} className='resButton' onClick={props.cancelReservation}>Cancel</button>
    </section>
  );
};

export default ReservationCard;