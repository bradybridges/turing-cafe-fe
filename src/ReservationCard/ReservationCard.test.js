import React from 'react';
import { shallow } from 'enzyme';
import ReservationCard from './ReservationCard';

describe('ReservationCard', () => {
  let wrapper = shallow( <ReservationCard 
    name='brady' 
    date='10/22' 
    time='10:30'
    number={10}
    id={1}
  /> );
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});