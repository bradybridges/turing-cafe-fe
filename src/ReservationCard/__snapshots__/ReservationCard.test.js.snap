// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCard should match the snapshot 1`] = `
<section
  className="resCard"
>
  <h2>
    brady
  </h2>
  <p>
    Date: 
    10/22
  </p>
  <p>
    Time: 
    10:30
  </p>
  <p>
    Number of guests: 
    10
  </p>
  <button
    className="resButton"
    id={1}
  >
    Cancel
  </button>
</section>
`;
